android/build.gradle

```diff
@@ -1,9 +1,8 @@
-
 apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion 25
-    buildToolsVersion "25.0.1"
+    buildToolsVersion "28.0.3"
```

Comment:
can you remove `buildToolsVersion` ? I think it's optional these days, and I don't want to force people to upgrade to 28.0.3

Reply:
Sure. I Got it.